x-pack/platform/plugins/shared/stack_connectors/common/thehive/schema.ts

```diff
@@ -57,6 +57,8 @@ export const ExecutorSubActionCreateAlertParamsSchema = schema.object({
   severity: schema.nullable(schema.number({ defaultValue: TheHiveSeverity.MEDIUM })),
   tlp: schema.nullable(schema.number({ defaultValue: TheHiveTLP.AMBER })),
   tags: schema.nullable(schema.arrayOf(schema.string())),
+  template: schema.number(),
```

Comment:
This should be optional (nullable); otherwise, it may break rules with TheHive configured.

Reply:
Also, I think it is better to have keys (strings) instead of numbers. For users, using the API directly would make it easier to figure out which template they can use.